test/EventHandlers/CLPool/CLPoolMintLogic.test.ts

```diff
@@ -0,0 +1,249 @@
+import { expect } from "chai";
+import type {
+  CLPool_Mint_event,
+  LiquidityPoolAggregator,
+  Token,
+} from "generated";
+import { processCLPoolMint } from "../../../src/EventHandlers/CLPool/CLPoolMintLogic";
+
+describe("CLPoolMintLogic", () => {
+  describe("processCLPoolMint", () => {
+    it("should create entity and calculate liquidity updates for successful mint", () => {
+      // Mock event data
+      const mockEvent: CLPool_Mint_event = {
+        params: {
+          sender: "0x1111111111111111111111111111111111111111",
+          owner: "0x2222222222222222222222222222222222222222",
+          tickLower: 100000n,
+          tickUpper: 200000n,
+          amount: 1000n,
+          amount0: 750n,
+          amount1: 500n,
+        },
+        srcAddress: "0x3333333333333333333333333333333333333333",
+        transaction: {
+          hash: "0x4444444444444444444444444444444444444444444444444444444444444444",
+        },
+        block: {
+          timestamp: 1000000,
+          number: 123456,
+          hash: "0x5555555555555555555555555555555555555555555555555555555555555555",
+        },
+        chainId: 10,
+        logIndex: 1,
+      };
```

Comment:
if the `mockEvents` used in this file are same - lets declare it once and reuse to reduce lines of code 👍